docs/docs/quickstart/index.md

```diff
@@ -0,0 +1,202 @@
+    operations:
+      - name: say
+        params:
+          - { name: msg, type: Message }
+          - { name: when, type: When }
+        return:
+          type: int
+    signals:
+      - name: justSaid
+        params:
+          - { name: msg, type: Message }
+enums:
+  - name: When
+    members:
+      - { name: Now, value: 0 }
+      - { name: Soon, value: 1 }
+      - { name: Never, value: 2 }
+structs:
+  - name: Message
+    fields:
+      - { name: content, type: string }
+```
+
+## 4. Generate code
+
+For the code generation we assume that both *ApiGear* files reside in an `apigear` subfolder next to the *Unreal Engine* project.
+In this case the folder structure should look similar to this.
+```bash
+📦ue_docs_example_project
+ ┣ 📂apigear
+ ┃ ┣ 📜helloworld.solution.yaml
+ ┃ ┗ 📜helloworld.module.yaml
+ ┣ 📂ue_docs
+ ┃ ┣ 📂Config
+ ┃ ┣ 📂Content
+ ┃ ┣ 📂Platforms
+ # highlight-next-line
+ ┃ ┣ 📂Plugins
+ ┃ ┣ 📂Source
+ ┃ ┗ 📜ue_docs.uproject
+```
+With the solution file from the previous paragraph the code will be generated in the `ue_docs/Plugins` folder.
+### Generate via CLI
+
+The following snippet shows how to generate code using the CLI.
+
+```bash
+$ apigear generate solution apigear/helloworld.solution.yaml 
+17:52:20 INF generated 92 files in 30ms. (33 write, 0 skip, 59 copy) topic=gen
+```
+* `generate` tells the CLI that it should generate code
+* `solution` specifies that we want to run a solution file
+  
+
+### Generate via Studio
+
+1. Open the project
+2. Go to the `Solutions` tab
+3. Click `Run` on the `helloworld.solution.yaml` entry
+
+
+<Figure caption="Generate code" src="/img/quick-start/apigear-studio-generate-code.png" />
+
+## 5. How to use the plugin
+
+The generated code can be used with *Blueprints* and *C++*. The following paragraphs show how this can be achieved.
+Make sure to have the generated plugin code in your projects plugins directory as noted in the [previous section](#4-generate-code).
+### Blueprint
+
+In *Blueprint* we first create an object of the type *Io World Hello*. This can then be  directly used or stored in a variable.
+In the screenshot we use the *asynchronous say* function to say "Hello world".
+
+<Figure caption="Hello interface blueprint" src="/img/quick-start/quick-start-blueprint.png" />
+
+### C++ 
+
+Since the generated interface is probably used outside of its own module, we have to add the module identifier to the list of dependency modules in the dependent module e.g. your project's or another plugin's module.
+This can be done like this, in the example *IoWorld* is the demo module's name:
+```cs showLineNumbers title="ue_docs.Build.cs"
+PublicDependencyModuleNames.AddRange(new string[] { "Core", "CoreUObject", "Engine", "InputCore", "IoWorld" });
+```
+
+Afterwards, we can simply include the header files for the `api` interface, the `stub` implementation and use it.
+The following code is an example and should not be used like this in production code.
```

Comment:
What shouldn't be used? The code itself? Or the style? A general disclaimed could maybe be written at the top of the quick-start guide.

Reply:
Removed the whole sentence. I think we can assmue that customers who write UE C++ code know what they do.
And there is the warning hint at the buttom for more information in the UE docs.